lighthouse-core/test/fraggle-rock/config/config-test.js

```diff
@@ -63,8 +63,8 @@ describe('Fraggle Rock Config', () => {
   });
 
   it('should throw on invalid artifact definitions', () => {
-    const configJson = {artifacts: [{id: 'FullPageScreenshot', gatherer: 'full-page-screenshot'}]};
-    expect(() => initializeConfig(configJson, {gatherMode})).toThrow(/FullPageScreenshot gatherer/);
+    const configJson = {artifacts: [{id: 'ScriptElements', gatherer: 'script-elements'}]};
```

Comment:
We are running out of candidates for this hehe

Reply:
yeah we are 😎